### ISSUE TITLE
[Web] PointerScrollEvent is returning values 5x higher than Desktop

### ISSUE THREAD

**Issue by esDotDev** [flutter/flutter#76913]
PointerScrollEvent.scrollDelta.y, we're seeing values of 100 on web, and 20 on Desktop. 

Reproduction: Use the middle-mouse scrollwheel, on dev branch, with code like:
```
Listener(
      onPointerSignal: (signal) {
        if (signal is PointerScrollEvent) {
          print("${signal.scrollDelta.dy}"); //  returns 100 on Web, 20 on Windows
        }
      },
```

[√] Flutter (Channel dev, 1.27.0-8.0.pre, on Microsoft Windows [Version 10.0.19042.685], locale en-US)
    • Flutter version 1.27.0-8.0.pre at C:\_dev\sdks\flutter
    • Framework revision b7d4806243 (7 days ago), 2021-02-19 09:22:45 -0800
    • Engine revision 6993cb229b
    • Dart version 2.13.0 (build 2.13.0-30.0.dev)

**Comment by esDotDev** [comment]
@yjbanov We can work around this easily enough, but this might be a quick fix?

**Comment by TahaTesser** [comment]
Hi @esDotDev 
Thanks for filing the issue 


<details>
<summary>code sample</summary>

```dart
import 'package:flutter/gestures.dart';
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Material App',
      theme: ThemeData.dark(),
      home: Home(),
    );
  }
}

class Home extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('Material App Bar'),
      ),
      body: Center(
        child: Listener(
          onPointerSignal: (signal) {
            if (signal is PointerScrollEvent) {
              print(
                  "${signal.scrollDelta.dy}"); //  returns 100 on Web, 20 on Windows
            }
          },
          child: FlutterLogo(
            size: 300,
          ),
        ),
      ),
      floatingActionButton: FloatingActionButton(
        child: Icon(Icons.add),
        onPressed: () {},
      ),
    );
  }
}

```

</details>


### Web
<details>
<summary>logs</summary>

```bash
Launching lib/main.dart on Chrome in debug mode...
 lib/main.dart
Debug service listening on ws://127.0.0.1:56194/VTHFH3rHiLw=/ws
Running with unsound null safety
For more information see https://dart.dev/null-safety/unsound-null-safety
Connecting to VM Service at ws://127.0.0.1:56194/VTHFH3rHiLw=/ws
-2.0001220703125
-21.64276123046875
-88.32275390625
-118.61541748046875
-133.84124755859375
-148.939208984375
-160.88104248046875
2.0001220703125
16.83746337890625
71.7315673828125
110.8709716796875
272.3272705078125
352.052001953125
2.0001220703125
19.98260498046875
84.051513671875
116.617431640625
150.05279541015625
339.35760498046875
378.77655029296875
2.0001220703125
21.60369873046875
83.9788818359375
116.6204833984375
282.70477294921875
162.45941162109375
-2.0001220703125
-21.66351318359375
-88.39813232421875
-118.61175537109375
-304.521484375
-175.6658935546875
-181.6314697265625
-2.0001220703125
-21.6326904296875
-88.28857421875
-118.5888671875
-286.2860107421875
-176.737060546875
-181.62689208984375
-2.0001220703125
-21.66107177734375
-88.3465576171875
-254.2840576171875
-150.66864013671875
-337.752685546875
-2.0001220703125
-21.6265869140625
-88.2952880859375
-118.57879638671875
-304.4232177734375
-175.635986328125

```

</details>

### Desktop

<details>
<summary>logs</summary>

```bash
Launching lib/main.dart on macOS in debug mode...
 lib/main.dart
Connecting to VM Service at ws://127.0.0.1:56086/WzF5Wh6OgAQ=/ws
flutter: -1.00006103515625
flutter: -10.00030517578125
flutter: -37.848968505859375
flutter: -55.40374755859375
flutter: -63.34228515625
flutter: -67.83279418945312
flutter: 1.00006103515625
flutter: 6.934356689453125
flutter: 26.140594482421875
flutter: 41.9732666015625
flutter: 56.50726318359375
flutter: 1.00006103515625
flutter: 9.20074462890625
flutter: 37.87628173828125
flutter: 54.488677978515625
flutter: 1.00006103515625
flutter: 9.204864501953125
flutter: 37.8924560546875
flutter: 54.493255615234375
flutter: 61.589202880859375
flutter: -1.00006103515625
flutter: -10.014495849609375
flutter: -42.0697021484375
flutter: -57.355804443359375
flutter: -65.125732421875
flutter: -71.94915771484375
flutter: -1.00006103515625
flutter: -10.82183837890625
flutter: -42.0184326171875
flutter: -58.311614990234375
flutter: -66.01119995117188
flutter: -71.0992431640625
flutter: -1.00006103515625
flutter: -10.815887451171875
flutter: -39.9365234375
flutter: -56.37420654296875
flutter: -61.586761474609375
flutter: -1.00006103515625
flutter: -9.203948974609375
flutter: 2.227935791015625
flutter: 12.931976318359375
flutter: 47.975921630859375
flutter: 61.067962646484375
flutter: 68.53683471679688
flutter: 1.00006103515625
flutter: 10.828094482421875
flutter: 42.055206298828125
flutter: 58.315887451171875
flutter: 66.92840576171875
flutter: 73.61709594726562
flutter: 1.00006103515625
flutter: 9.998931884765625
flutter: 37.8546142578125
flutter: 56.363525390625
flutter: 63.339691162109375
flutter: 1.00006103515625
flutter: 9.99969482421875
flutter: 42.031707763671875
flutter: 57.334747314453125
flutter: 66.01089477539062
flutter: 71.10015869140625
flutter: 1.00006103515625
flutter: 10.826568603515625
flutter: 42.051544189453125
flutter: 58.311004638671875
flutter: -1.00006103515625
flutter: -9.994354248046875
flutter: -39.932403564453125
flutter: -56.372833251953125
flutter: 1.00006103515625
flutter: 10.8203125
flutter: 44.139404296875
flutter: 59.29473876953125
flutter: 66.91207885742188
flutter: 1.00006103515625
flutter: 9.997100830078125
flutter: 42.041778564453125
flutter: 57.349090576171875
flutter: 66.02035522460938
flutter: -1.00006103515625
flutter: -10.82550048828125
flutter: -44.195709228515625
flutter: -59.308929443359375
flutter: -66.91604614257812
flutter: -73.61236572265625
flutter: -1.00006103515625
flutter: -10.825347900390625
flutter: -42.025909423828125
flutter: -58.3221435546875
flutter: -66.92794799804688
flutter: -73.61083984375
flutter: -1.00006103515625
flutter: -9.992218017578125
flutter: -42.041473388671875
flutter: -58.321990966796875
flutter: -66.00387573242188
flutter: -72.7764892578125
flutter: 1.00006103515625
flutter: 10.817413330078125
flutter: 42.060699462890625
flutter: 58.323822021484375
flutter: 66.9329833984375
flutter: 73.62472534179688
flutter: 1.00006103515625
flutter: 10.8160400390625
flutter: 44.1748046875
flutter: 59.309844970703125
flutter: 67.84530639648438
flutter: 75.32943725585938
flutter: 81.240234375

```

</details>


<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel beta, 1.26.0-17.8.pre, on macOS 11.2.2 20D80 darwin-x64, locale en-GB)
    • Flutter version 1.26.0-17.8.pre at /Users/tahatesser/Code/flutter_beta
    • Framework revision 044f2cf560 (4 days ago), 2021-02-24 13:02:05 -0800
    • Engine revision 042c82b02c
    • Dart version 2.12.0 (build 2.12.0-259.16.beta)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.
[!] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
[✓] VS Code (version 1.53.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0
[✓] Connected device (2 available)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios            • iOS 14.4
    • Chrome (web)         • chrome                    • web-javascript • Google Chrome 88.0.4324.192
! Doctor found issues in 2 categories.
```
```bash
[✓] Flutter (Channel dev, 1.27.0-8.0.pre, on macOS 11.2.2 20D80 darwin-x64, locale en-GB)
    • Flutter version 1.27.0-8.0.pre at /Users/tahatesser/Code/flutter_dev
    • Framework revision b7d4806243 (10 days ago), 2021-02-19 09:22:45 -0800
    • Engine revision 6993cb229b
    • Dart version 2.13.0 (build 2.13.0-30.0.dev)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.
[!] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
[✓] VS Code (version 1.53.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0
[✓] Connected device (3 available)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios            • iOS 14.4
    • macOS (desktop)      • macos                     • darwin-x64     • macOS 11.2.2 20D80 darwin-x64
    • Chrome (web)         • chrome                    • web-javascript • Google Chrome 88.0.4324.192
! Doctor found issues in 2 categories.
```
```bash
[✓] Flutter (Channel master, 1.27.0-9.0.pre.128, on macOS 11.2.2 20D80 darwin-x64, locale en-GB)
    • Flutter version 1.27.0-9.0.pre.128 at /Users/tahatesser/Code/flutter_master
    • Framework revision 199d5dc086 (10 hours ago), 2021-02-28 14:36:02 -0800
    • Engine revision 34236c9837
    • Dart version 2.13.0 (build 2.13.0-81.0.dev)
[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.
[!] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome
[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
[✓] VS Code (version 1.53.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0
[✓] Connected device (3 available)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios            • iOS 14.4
    • macOS (desktop)      • macos                     • darwin-x64     • macOS 11.2.2 20D80 darwin-x64
    • Chrome (web)         • chrome                    • web-javascript • Google Chrome 88.0.4324.192
! Doctor found issues in 2 categories.
```

</details>

**Comment by ferhatb** [comment]
@esDotDev 
The delta value is typically 120 per notch. [See](https://developer.mozilla.org/en-US/docs/Web/API/Element/mousewheel_event). 

@cbracken  : It would benefit to align desktop and web values. What is the desktop one based on ?

**Comment by huycozy** [comment]
This issue can be reproducible on latest `stable` and `master` channel with provided sample code.
The values are different from the above comments but overall they are not same  between platforms.

- On the Apple device environments, there is only difference between Sarafi and others:
    - MacOS: 13.0
    - Chrome: 13.0
    - Safari: 6.5
- Windows: 20.0

<details>
<summary>flutter doctor -v</summary>

```bash
[✓] Flutter (Channel stable, 2.10.5, on macOS 12.2.1 21D62 darwin-x64, locale en-VN)
    • Flutter version 2.10.5 at /Users/huynq/Documents/GitHub/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5464c5bac7 (2 weeks ago), 2022-04-18 09:55:37 -0700
    • Engine revision 57d3bac3dd
    • Dart version 2.16.2
    • DevTools version 2.9.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] Android Studio (version 4.1)
    • Android Studio at /Users/huynq/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/201.7042882/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] Android Studio
    • Android Studio at /Users/huynq/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-1/203.7185775/Android Studio Preview.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)

[✓] IntelliJ IDEA Community Edition (version 2020.3.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.62.3)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.28.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 100.0.4896.127

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```

```bash
[✓] Flutter (Channel master, 2.13.0-0.0.pre.866, on macOS 12.2.1 21D62 darwin-x64, locale en-VN)
    • Flutter version 2.13.0-0.0.pre.866 at /Users/huynq/Documents/GitHub/flutter_master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 6246a09d0e (42 minutes ago), 2022-05-03 00:04:07 -0400
    • Engine revision a00140cb28
    • Dart version 2.18.0 (build 2.18.0-78.0.dev)
    • DevTools version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/huynq/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • ANDROID_HOME = /Users/huynq/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] Android Studio (version 4.1)
    • Android Studio at /Users/huynq/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-0/201.7042882/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[!] Android Studio
    • Android Studio at /Users/huynq/Library/Application Support/JetBrains/Toolbox/apps/AndroidStudio/ch-1/203.7185775/Android Studio Preview.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    ✗ Unable to find bundled Java version.
    • Try updating or re-installing Android Studio.

[✓] IntelliJ IDEA Community Edition (version 2020.3.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.62.3)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.28.0

[✓] Connected device (2 available)
    • macOS (desktop) • macos  • darwin-x64     • macOS 12.2.1 21D62 darwin-x64
    • Chrome (web)    • chrome • web-javascript • Google Chrome 100.0.4896.127

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 1 category.
```
</details>

**Comment by cbracken** [comment]
Agreed that we want to get these consolidated between web and desktop. IIRC our desktop values were actually based off Chromium's values, but I'll take a look for each platform and update here:

* macOS: https://github.com/flutter/engine/blob/87226dbd54efa4f37ea6b09488259795b312fded/shell/platform/darwin/macos/framework/Source/FlutterViewController.mm#L701
* Windows:
* Linux:

**Comment by esDotDev** [comment]
As a developer, I almost always apply some multiple to this value anyways, so would be more or less happy with a -1 or 1 for each click of the wheel.  

The main thing is that it must be consistent across platforms.

**Comment by Piinks** [comment]
I think the intention here is to match native implementations, in general we avoid arbitrary configurations when most folks expect the scrolling feel of the platform they are on.

We could explore ways to allow the user to configure the delta if they want something different. 

@ferhatb's reference above is pretty thorough, and indicates mouse wheel deltas are not the same on every platform and browser, so it appears to be expected.

**Comment by esDotDev** [comment]
It makes sense for Flutter to handle that sort of abstraction in it's own internal lists etc, but imo doesn't make sense to let that un-normalized value leak out into the mouse wheel event listeners which are used for more generic purposes. 

Just consider a simple use case where we want to zoom in on scroll. This is not traditional "scrolling" and wouldn't follow any of the platform specific rules you're implementing. 

With this approach, Developer can not do: `scale += scroll.delta * scrollFactor` becuase they will get wildly different results. Instead they need to normalize it manually.

Normalizing it is not really the issue though, the issue is that this is non-obvious and counter-intuitive, so you won't know you have to normalize, until suddenly someone logs a bug that zoom is extremely fast on web (which is what happened to us). 

It's easy to say "You should be testing on web", but when you're supporting 6 platforms, its much more time efficient if we just get standardized / normalized behavior across platforms because there is less magic, which means less surprises to the cross-platform developer.

**Comment by Piinks** [comment]
Perhaps in addition to `PointerScrollEvent.scrollDelta`, which is intended for scrolling, we can have a `PointerScrollEvent.normalizedScrollDelta`?

I think it is important to preserve the correct delta in the scroll event, otherwise we would need to implement the platform specific behavior for every widget that interacts with this information. This would be more bug prone, more difficult to maintain, and have a higher chance of widgets getting left out and being inconsistent.

**Comment by esDotDev** [comment]
Ya that makes sense, and the presence of the normalized field name indicates that the other field is not normalized so it's a little more self documenting.

**Comment by Piinks** [comment]
As we explore this more, I just thought of trackpad scrolling (which also includes the Apple Magic Mouse). We should make sure this input device is included in testing as it also contributes PointerScrollEvents. 
Full support for trackpads in currently in progress (#23604), AFAIK it is nearly completed so once that is done we should be able to fully understand how to do this cleanly for all pointer scrolling. :)

**Comment by moffatman** [comment]
There was some change recently in Windows to respect the system "lines per scroll" setting. I think the values should be much closer now, can you check and see if it's OK now? @esDotDev

**Comment by gspencergoog** [comment]
@Piinks Should this still be P1?  And if so, would you be willing to be assigned to this? (Desktop Triage)

**Comment by Piinks** [comment]
It looks like the last update suggested it may be fixed. I can follow up on checking that.

**Comment by Piinks** [comment]
Looks like we are spot on for Mac, checking on Linux and Windows with some help just to cover all the bases.

**Comment by Piinks** [comment]
Confirmed we are matching windows versus web on windows as well. 🎉 
One more to go.

**Comment by Piinks** [comment]
I've recruited some help for Linux from @cbracken (thank you!), leaving instructions here to validate it is fixed on that platform:

1. Take this app: https://gist.github.com/Piinks/06166fc4dd02f77a2775831dc7a11c3e
2. Check out the framework at tip of tree.
3. Run app on Linux, scroll wheel, observe values in console.
4. Run app on web (on Linux), scroll wheel, observe values in console.

If they are about the same, this can be closed! 🎉 

It can be finnicky trying to get the mouse scroll wheel to scroll exactly the same way each time, which is why I say `about the same`. In the OP, the difference was significant, 100 versus 20.

**Comment by esDotDev** [comment]
fwiw, there is also this issue relating to android + bluetooth keyboard
https://github.com/flutter/flutter/issues/82973

**Comment by Piinks** [comment]
Thanks @esDotDev! That is a separate issue that I was just digging into yesterday. We can leave it open an continue to follow it there. :)

**Comment by gspencergoog** [comment]
Reassigning back to @cbracken to answer https://github.com/flutter/flutter/issues/76913#issuecomment-1673833124
